linux/shim/service.go

```diff
@@ -211,6 +212,7 @@ func (s *Service) Resume(ctx context.Context, r *shimapi.ResumeRequest) (*google
 }
 
 func (s *Service) Exit(ctx context.Context, r *shimapi.ExitRequest) (*google_protobuf.Empty, error) {
+	unix.Unmount(filepath.Join(s.path, "rootfs"), 0)
```

Comment:
I think this is probably ok. I will look at CI for Darwin, I commented on https://github.com/containerd/containerd/issues/866